tatus

```diff
@@ -0,0 +1,176 @@
+[1mdiff --git a/app/src/androidTest/java/sdp/moneyrun/GameTest.java b/app/src/androidTest/java/sdp/moneyrun/GameTest.java[m
```

Comment:
What is this?

Reply:
No idea, must have been generated automatically by AndroidStudio for one of its services. I'll remove it